types/schemas/enforcement/enums/BreachTypes.ts

```diff
@@ -0,0 +1,48 @@
+// Sourced from https://editor.planx.uk/open-digital-planning/report-a-planning-breach-about-the-breach
+//   and https://editor.planx.uk/open-digital-planning/report-a-planning-breach-scoping
+export const BreachTypes = {
+  alter: 'Alteration',
+  'alter.boundary': 'New or modified boundary fence or wall',
+  'alter.equipment.antennae': 'Microwave antennas or satellite dishes',
+  'alter.highways': 'Creation of a new access (eg a dropped kerb)',
+  'alter.landscape': 'Change of land levels',
+  'alter.openings': 'Works to windows or doors',
+  'alter.roof': 'Changes to roof including skylights',
+  'alter.soilPipes': 'Chimney flue or soil and vent pipes',
+  'alter.surfaces.parking': 'Creation of driveway or laying of hardstanding',
+  'alter.swimmingPool': 'New or modified swimming pool',
+  'alter.trees': 'Works to trees',
+  'alter.trees.tree': 'Works to a dangerous or dead tree',
+  'alter.trees.tree.other': 'Other works to a tree',
+  'alter.trees.tree.prune': 'Works to a healthy tree',
+  'alter.trees.tree.remove': 'Felling of a tree',
+  changeOfUse: 'Unauthorised change of use',
+  'changeOfUse.caravans': 'Caravan or mobile homes',
+  extend: 'Extension',
+  'extend.porch': 'New or modified porch',
+  'extend.roof.dormer': 'Inserting a dormer window',
+  listed: 'Works to a listed building',
+  minerals: 'Minerals and waste development',
+  other: 'Other type of planning breach',
+  outbuilding: 'New or modified outbuilding (eg garage or shed)',
+  planningCondition: 'Breach of a planning condition',
+  signage: 'New adverts or signage (including flags)',
+  untidySite: 'Untidy land',
```

Comment:
In the actual flow, these "option" data values are all confusingly named/prefixed like `proposal.projectType.breach.alter.boundary` - we'll want to strip the `proposal.projectType.breach` prefix when mapping the Planx passport to payload